### PR TITLE
Misc form fix/imp

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

File: cms_form/models/cms_form_mixin.py [hunk 1]
```diff
@@ -324,15 +324,23 @@ def form_get_request_values(self):
...
+        res = {}
+        for k, v in _values.items():
+            if k in ('csrf_token', ):
+                continue
+            if k.endswith(':list'):
+                # fields w/ multiple values
+                # TODO
+                # 1. add test and docs
+                # 2. support more "transformers" (`:int` for instance)
+                v = _values.getlist(k)
```
Comment (tschanzt) [inline cms_form/models/cms_form_mixin.py:336]: i think we should skip this if we have already processed an item of this list.

**Review by tschanzt — CHANGES_REQUESTED** [review]
one small thing. also travis is failing.

File: cms_form/models/cms_form_mixin.py (hunk 1 above)
Comment (simahawk) [reply cms_form/models/cms_form_mixin.py:336]: the funny thing is that "dictionary-wise" you get only one key, so if you iter or access the key you get only one. That's why you must use getlist. See http://werkzeug.pocoo.org/docs/0.14/datastructures/#werkzeug.datastructures.MultiDict ;)

Comment (tschanzt) [reply cms_form/models/cms_form_mixin.py:336]: Interesting, didn't know that.

**Review by tschanzt — APPROVED** [review]

**Pull request merged**